sp800-63c/sec10_usability.md

```diff
@@ -53,19 +55,19 @@ It is possible that a user may maintain multiple accounts at a single IdP, or in
 4. Enable users to consent to a partial list of attributes, rather than an all or nothing approach.  Similarly, RPs are encouraged to allow some degree of online access even if the user does not consent to share all information.
 5. Base attribute sharing on the transaction being requested, not on all possible transactions a user may or may not access at the RP.
-6. Do not display system generated attributes such as a pairwise pseudonymous identifier (not to be confused with system derived attribute claims, such as 'older than' derived from 'date of birth'), to users, even if it is shared with the RP as part of the authentication response.  If these types of attributes are visible to the user, display them in a portion of the UI that is separated from actual user attributes such as name or email. Additionally, provide a user-friendly description of the attributes to avoid confusion. Toggle this information on/off, via a UI control such as a check box, with the default as `off,' so the user has to explicitly select whether to review this information (see SP 800-53C section 6. Assertion Presentation and section 9. Privacy Considerations).
-7. Use standards, such as OAuth or OpenID Connect, that build attribute consent into the protocol—so they're not a feature external to the federated transaction.
-8. Provide easy to find redress methods (see [Section 6. Assertion Presentation](# #sec6) for more details) such that a user can recover from invalid attribute information claimed by the IdP.
-9. Limit the number of times a user will need to consent to the release of attributes. Upon first access it is normal to request consent; on subsequent access attempts to the same RP, consent is not needed, provided the IdP offers appropriate lifecycle management functions as discussed in [10.1.4 Lifecycle Management].
+6. Do not display system generated attributes such as a pairwise pseudonymous identifier (not to be confused with system derived attribute claims, such as 'older than' derived from 'date of birth'), to users, even if it is shared with the RP as part of the authentication response.  If these types of attributes are visible to the user, display them in a portion of the UI that is separated from actual user attributes such as name or email. Additionally, provide a user-friendly description of the attributes to avoid confusion. Toggle this information on/off, via a UI control such as a check box, with the default as `off,' so the user has to explicitly select whether to review this information (see SP 800-53C [Section 6](#sec6) and [Section 9](#sec9)).
```

Comment:
The backquote (grave accent mark) following "with the default as" tells Markdown that a preformatted text block begins. This should be a simple ' (apostrophe) character.